linode/lkenodepool/framework_resource_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build integration
+//go:build (integration || lkenodepool) && !optional && !long_running
```

Comment:
Do we still want to have the `long_running` tag? It seems not being addressed/described after the clean up.